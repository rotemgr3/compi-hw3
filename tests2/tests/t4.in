//piazza 65 - should be errorUndef +
void main() {
	int a = 5;
	int d = 7;
	if (a>g) return;  //returns error bc g is not defined
}

