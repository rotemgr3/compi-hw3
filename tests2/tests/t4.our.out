line 5: variable g is not defined
